typescript/packages/common-ui/src/components/checkbox.ts

```diff
@@ -0,0 +1,13 @@
+import { view } from "../hyperscript/render.js";
+import { eventProps } from "../hyperscript/schema-helpers.js";
+
+export const checkbox = view(
```

Comment:
Would express this as:

```
export const input = view(
  "input",
  {
    ...eventProps(),
    type: { type: { anyOf: ["text", "checkbox"] } }
    id: { type: "string" },
    checked: { type: "boolean" },
    value: { type: "string" },
  }
);
```

Input is a weird edge case because the properties are disjoint from the tag type, and the type property acts as a flag that customizes the input properties. We don't currently have a way to express tags that are disjoint from their properties like that.

Reply:
Ah, I see you do this in `input.ts`. Suggest deleting this file.